.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: CI
+on:
+  push:
+    paths-ignore:
+        - '*.md'
+  pull_request:
+    paths-ignore:
+        - '*.md'
+jobs:
+  test:
+    name: ${{ matrix.node-version }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [10, 12, 14, 16]
+    steps:
+      - uses: actions/checkout@v2.3.4
```

```suggestion
      - uses: actions/checkout@v2
```

Reduces the amount of servicing required